components/post_view/reaction/reaction.test.tsx

```diff
@@ -5,17 +5,23 @@ import React from 'react';
 import {shallow} from 'enzyme';
 
 import Reaction from 'components/post_view/reaction/reaction';
+import {TestHelper} from 'utils/test_helper';
+import {Reaction as ReactionType} from 'mattermost-redux/types/reactions';
```

Comment:
Can you move this to above components imports ?

Reply:
Done